cmd/crank/update.go

```diff
@@ -38,16 +39,33 @@ type updateConfigCmd struct {
 }
 
 // Run runs the Configuration update cmd.
-func (c *updateConfigCmd) Run(k *kong.Context) error { // nolint:gocyclo
-	kube := typedclient.NewForConfigOrDie(ctrl.GetConfigOrDie())
+func (c *updateConfigCmd) Run(k *kong.Context, logger logging.Logger) error { // nolint:gocyclo
+	logger = logger.WithValues("Name", c.Name)
+	kubeConfig, err := ctrl.GetConfig()
+	if err != nil {
+		logger.Info(errKubeConfig, "error", err)
+		return errors.Wrap(err, errKubeConfig)
+	}
+	logger.Info("Found kube config")
+	kube, err := typedclient.NewForConfig(kubeConfig)
+	if err != nil {
+		logger.Info(errKubeClient, "errors", err)
```

Comment:
I notice you are using the key `errors` in some places and `error` in others, is there a specific reason for each?